tools/ci_build/rewriter.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import os
+import shutil
+import onnx
+from onnx import AttributeProto as AP
+
+#pylint: disable=no-member,too-many-locals,too-many-statements
+
+def extract_ops_from_file(file_path, referred_ops):
+    '''extract ops from csv file with format:
+        op_type,domain,opset'''
+
+    if not os.path.isfile(file_path):
+        return referred_ops
+
+    with open(file_path, 'r') as csv_to_read:
+
+        for line in csv_to_read.readlines():
+            op_type, domain, raw_opset = line.strip().split(',')
```

Comment:
This requires the user to provide a lot of lines of input which will be painful to create or to edit. Can we consider something like the following so it's one line per domain+opset?

domain,opset,op1[,op2...]

Reply:
Good idea while that compromise the readability and adding the complexity of script deciphering it.

Reply:
Using csv is an arbitrary choice and afaik not a hard requirement. We just need a reasonably simple way to provide the list of things to enabled. Could also use a different separator between the main pieces and the individual ops. Not sure it makes it less readable or complex. 

```
input = "ai.onnx:10:cast,indentity,add"
domain, opset, ops = input.split(':')
ops = ops.split(',')
```

Reply:
It is a good one but wouldn't csv or tsv sounds more generic for people in the area?
Further, with multiple sects in each line isn't it easier to make mistakes?
If we want to structure the file, json is the option but takes more time for users to compose one compared with a simple csv.

Reply:
If we plan to add types later on, the format should take into account that as well.

Reply:
Agreed. So if have format be like:
domain,opset,op
For each line, it is could be easily extend to
domain,opset,op,type

Reply:
Or let's just go with json?

Reply:
Switched to json.